src/metabase/util/yaml.clj

```diff
@@ -1,27 +1,63 @@
 (ns metabase.util.yaml
+  "Convenience functions for parsing and generating YAML."
   (:refer-clojure :exclude [load])
   (:require
+   [clj-yaml.core :as yaml]
+   [clojure.java.io :as io]
    [clojure.string :as str]
    [metabase.util :as u]
+   [metabase.util.date-2 :as u.date]
    [metabase.util.files :as u.files]
    [metabase.util.i18n :refer [trs]]
-   [metabase.util.log :as log]
-   [yaml.core :as yaml])
+   [metabase.util.log :as log])
   (:import
-   (java.nio.file Files Path)))
+   (java.nio.file Files Path)
+   (java.time.temporal Temporal)))
 
 (set! *warn-on-reflection* true)
 
+(defn- vectorized
```

Comment:
I plan on opening a PR against `clj-yaml` to make this unnecessary. I just have to sneak a single `v` into the code...